packages/components/src/Banner/Banner.tsx

```diff
@@ -74,12 +85,37 @@ export function Banner({
     [styles.dismissibleSpacing]: dismissible,
   });
 
+  function getPrev() {
+    current > 0 && setCurrent((current - 1) % messages.length);
+  }
+  function getNext() {
+    setCurrent((current + 1) % messages.length);
+  }
+
   return (
     <>
       {showFlash && (
         <div className={flashClassNames} ref={bannerRef} role="status">
           <div className={contentClassNames}>
-            <Text>{children}</Text>
+            {!multiple && <Text>{children}</Text>}
+            {multiple && (
+              <>
+                <div className={styles.message}>
+                  <Text>{messages[current]}</Text>
+                </div>
+                <div className={styles.pagination}>
+                  <div className={styles.prev} onClick={getPrev}>
+                    <Icon name="arrowLeft" />
+                  </div>
+                  <Text>
+                    {current + 1} of {messages.length}
+                  </Text>
+                  <div className={styles.next} onClick={getNext}>
+                    <Icon name="arrowRight" />
+                  </div>
```

Comment:
I'm a noob, but could this little arrow guy be its own component? It's really cool!